src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -31,9 +31,19 @@ namespace NUnit.Framework.Constraints
     /// item in a collection, succeeding only if a specified
     /// number of items succeed.
     /// </summary>
-    public class ExactCountConstraint : PrefixConstraint
+    public class ExactCountConstraint : Constraint
     {
-        private int expectedCount;
+        private readonly int expectedCount;
+        private readonly IConstraint itemConstraint;
```

Comment:
If it's made a resharper setting, does that mean that new contributors who
use resharper will see tons of warnings? Do we want that to happen?

2016-09-02 11:49 GMT-07:00 Rob Prouse <notifications@github.com>:

> In src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs
> <https://github.com/nunit/nunit/pull/1780#discussion_r77395223>:
>
> >      {
> > -        private int expectedCount;
> > +        private readonly int expectedCount;
> > +        private readonly IConstraint itemConstraint;
>
> If that is a Resharper option, feel free to add it as a part of this PR or
> as a separate PR if you prefer. I stopped using Resharper so stopped
> keeping the dotsettings up to date.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/nunit/nunit/pull/1780/files/2ac66a889944faa53a90b2515299a0d7fa47801c#r77395223>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACjfhU-IXzXcBQQ28nBqU7yJ2tCDAThZks5qmG-7gaJpZM4JzPJB>
> .
>